### DISCUSSION TITLE
When I open multiple sheets in the terminal, why do they have to be visited to have an alt-shortcut?

### DISCUSSION THREAD

**Discussion by reagle** [saulpw/visidata#1469] [Q&A] [answered]
`> vd a.csv b.csv c.vsd`

Only a.csv has an `alt+1` shortcut. If I select the 3rd sheet, it then gets assigned `alt+2`. Couldn't they all be assigned from the start using the argument order from the terminal?

<img width="396" alt="Screen Shot 2022-08-10 at 12 40 14" src="https://user-images.githubusercontent.com/596814/183966047-db032986-3a52-4f30-997c-078b76e47d74.png">

**Comment by anjakefala** [comment, marked as answer]
VisiData does not load sheets until they are visited. This is to avoid an upfront memory demand for potentially really large amounts of data.

I believe the keystroke is assigned when the sheet is loaded, but you could create a feature request to have it assigned upon opening. We can look into it, that is a reasonable request.
